src/style/flatpickr.styl

```diff
@@ -226,6 +226,13 @@
         transition fill 0.1s
         fill inherit
 
+  &.disabled
+    &, &:hover
```

Comment:
This should go in your application's custom CSS.
The only thing here should be `display: none`